v4/source/webhooks.yaml

```diff
@@ -450,7 +450,7 @@
             schema:
               type: object
               required:
-                - id
+                - hook_id
```

Comment:
This is correct. It should be the opposite way. The json object field name is `id`. So we should update the path name to be `id` as well.